utils/test_all.sh

```diff
@@ -9,18 +9,28 @@
 # It starts deployment using templates, with different input (all templates to be used).
 # Supposed to be usefull to validate templates after they are changed.
 # Note: To check the templates, they must be uploaded to some public place. In this example it is github.
-# You need to set  baseUrl in parameters file according to the place where templates are uploaded.
-# Also you need to set AZURE_TEMPLATE_URI, e.g. you can add this line to your shell init file:
-# export AZURE_TEMPLATE_URI="https://raw.githubusercontent.com/ibekleiner/infoblox-azure-templates/master/main/mainTemplate.json"
-# Also please set AZURE_DEV_ID to avoid conflicts with developers running tests simultaneously:
-# export AZURE_DEV_ID="dk"
+# If you are using not the master brunch, You need to change according to the place where templates are uploaded:
+# baseUrl in parameters files and AZURE_TEMPLATE_URI in this test script.
+AZURE_TEMPLATE_URI="https://raw.githubusercontent.com/infobloxopen/infoblox-azure-templates/master/main/mainTemplate.json"
 
-RESOURCE_GROUP="${AZURE_DEV_ID}templtestgroup"
+# To avoid overlapping with other developers please change PREFIX in test script. Also you should
+# modify parameter files accordingly.
+# You can quickly do the job by running this command from the current directory:
+# find . -type f -exec sed -i 's/former_prefix/new_prefix/g' {} +
+PREFIX="templtest"
+
+RESOURCE_GROUP="${PREFIX}group"
 LOCATION="eastus"
-DEPLOYMENT_NAME="${AZURE_DEV_ID}newdeployment$(date +%Y%m%d%H%M%S)"
+DEPLOYMENT_NAME="${PREFIX}deployment$(date +%Y%m%d%H%M%S)"
 
 PARAMETERS_DIR="utils/params"
 
+if azure group show ${RESOURCE_GROUP} &> /dev/null; then
+    echo "Resource group ${RESOURCE_GROUP} exists. Please, change the prefix according to instructions in the script
```

Comment:
Would be nice to print the same instruction as in comments instead of making user to go into the script code.